File: src/main/java/org/example/model/Member.java
package org.example.model;
import java.time.LocalDate;
import java.util.Date;

public class Member extends Record{
    //дата напоминания
    Date dateMember;
    //по умолчанию новое напоминание
    StateMember state = StateMember.New;
    public void setDateMember(Date value){
        dateMember = value;
    }
    public void setStatusMember(StateMember value){
        state = value;
    }

    @Override
    public void createRecord(String text, LocalDate dateCreate) {
        super.createRecord(text, dateCreate, TypeRecord.Member, -1);
    }
}

File: src/main/java/org/example/model/Member.kt
package org.example.model

import java.time.LocalDate
import java.util.Date

class Member : Record() {
    //дата напоминания
     lateinit var dateMember: Date;
    //по умолчанию новое напоминание
    var state : StateMember = StateMember.New;

    fun setDateMember(value :Date){
        dateMember = value;
    }
    fun setStatusMember(value : StateMember){
        state = value;
    }

    override fun createRecord(text : String, dateCreate: LocalDate) {
        super.createRecord(text, dateCreate, TypeRecord.Member, -1);
    }
}